VelocidiSDK/VelocidiSDK/Tracking/Model/VSDKProductCustomization.h

```diff
@@ -0,0 +1,21 @@
+#import "VSDKTrackingEvent.h"
+@class VSDKProduct;
+@class VSDKCustomization;
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface VSDKProductCustomization : VSDKTrackingEvent
+
+/**
+  Product associated with this product customization
+ */
+@property NSMutableArray<VSDKProduct *> *product;
+
+/**
+  Customization associated with this product customization
+ */
+@property VSDKCustomization *productCustomization;
```

Comment:
This is not possible because the backend is expecting a `productCustomization` (see https://docs.velocidi.com/knowledgebase/tracking-api/#productCustomization)